dependencies.props

```diff
@@ -19,24 +19,33 @@
     <StandardCurrentRef>549ba26b2ed7c341a287c63b494e38496319802c</StandardCurrentRef>
   </PropertyGroup>
 
-  <!-- Auto-upgraded properties for each build info dependency. -->
+  <!-- Product dependency versions. -->
+  <PropertyGroup>
+    <NETStandardLibraryPackageVersion>2.0.1</NETStandardLibraryPackageVersion>
+    <NETStandardLibraryPackageId>NETStandard.Library</NETStandardLibraryPackageId>
+
+    <!-- SNI runtime package -->
+    <RuntimeWinX64RuntimeNativeSystemDataSqlClientSniPackageVersion>4.4.0</RuntimeWinX64RuntimeNativeSystemDataSqlClientSniPackageVersion>
+  </PropertyGroup>
+
+  <!-- Tests/infrastructure dependency versions. -->
   <PropertyGroup>
-    <PlatformPackageVersion>2.0.0</PlatformPackageVersion>
     <CoreFxExpectedPrerelease>servicing-25714-01</CoreFxExpectedPrerelease>
-    <CoreClrPackageVersion>2.0.2-servicing-25712-01</CoreClrPackageVersion>
-    <ExternalExpectedPrerelease>beta-25307-00</ExternalExpectedPrerelease>
+    <MicrosoftNETCorePlatformsPackageVersion>2.0.0</MicrosoftNETCorePlatformsPackageVersion>
+    <MicrosoftNETCoreRuntimeCoreCLRPackageVersion>2.0.2-servicing-25712-01</MicrosoftNETCoreRuntimeCoreCLRPackageVersion>
     <ProjectNTfsExpectedPrerelease>beta-25317-00</ProjectNTfsExpectedPrerelease>
     <ProjectNTfsTestILCExpectedPrerelease>beta-25317-00</ProjectNTfsTestILCExpectedPrerelease>
     <ProjectNTfsTestILCPackageVersion>1.0.0-beta-25317-00</ProjectNTfsTestILCPackageVersion>
-    <NETStandardPackageVersion>2.0.1</NETStandardPackageVersion>
-    <NETStandardPackageId>NETStandard.Library</NETStandardPackageId>
+    <MicrosoftNETCoreDotNetHostPackageVersion>2.0.1-servicing-25615-03</MicrosoftNETCoreDotNetHostPackageVersion>
+    <MicrosoftNETCoreDotNetHostPolicyPackageVersion>2.0.1-servicing-25615-03</MicrosoftNETCoreDotNetHostPolicyPackageVersion>
     <MicrosoftNETCoreAppPackageVersion>2.0.0-preview2-25317-03</MicrosoftNETCoreAppPackageVersion>
-    <!-- Use the SNI runtime package -->
-    <SniPackageVersion>4.4.0</SniPackageVersion>
-  </PropertyGroup>
 
-  <!-- Full package version strings that are used in other parts of the build. -->
-  <PropertyGroup>
+    <!-- CoreFX-built SNI identity package -->
+    <RuntimeNativeSystemDataSqlClientSniPackageVersion>4.4.0</RuntimeNativeSystemDataSqlClientSniPackageVersion>
+
+    <!-- Backward compatibility for BuildTools usage. -->
+    <PlatformPackageVersion>$(MicrosoftNETCorePlatformsPackageVersion)</PlatformPackageVersion>
```

Comment:
Looking deeper into this.  This won't work.  When `2.0.0` is hard-coded for `MicrosoftNETCorePlatformsPackageVersion`, this property will get `2.0.0`.  But then later when we import the `DotNetPackageVersionPropsPath` file, the first property will get updated, but then this one will be stuck with the hard-coded version.

Reply:
Good point, I'll move the alias after the import. I noticed this situation for other properties in an earlier prototype (where I didn't rename everything quite yet), but I forgot when adding this compat property later on.

Reply:
Submitted: https://github.com/dotnet/corefx/pull/24647